peachjam/views/generic_views.py

```diff
@@ -274,14 +277,14 @@ def get_notices(self):
         return []
 
     def prefix_images(self, document):
-        """Rewrite image URLs so that we can server them correctly."""
+        """Rewrite image URLs so that we can serve them correctly."""
         root = html.fromstring(document.content_html)
 
         for img in root.xpath(".//img[@src]"):
             src = img.attrib["src"]
             if not src.startswith("/") and not src.startswith("data:"):
                 img.attrib["src"] = (
-                    document.expression_frbr_uri + "/media/" + img.attrib["src"]
+                    document.expression_frbr_uri + "/" + img.attrib["src"]
```

Comment:
this must not change. The documents that have `content_html_is_akn == True` have `media/` already added, and so must not be processed through this function (which you have already added a guard for on line 170 above).

The documents that have `content_html_is_akn == False` MUST have this added because they don't prefix their images at all. This is for when images are embedded in Word documents.